src/components/Encoding.js

```diff
@@ -150,7 +162,18 @@ class Encoding extends Component<EncodingProps> {
             <Form horizontal className={classes.advanced}>
               <AdvancedFormGroup bsSize="xs">
                 <Col componentClass={ControlLabel} sm={3}>
-                  Type:
+                  <OverlayTrigger
```

Comment:
it seems like these could be generated via a function that is invoked at each position and passed the property to display. something like:
```
renderLabel = (label: string, value: number) => {
   return <OverlayTrigger ... 
}
```

and a sample invocation could be `renderLabel('zero', overlayDescriptions.zero)`

Reply:
yeah that's a good idea, i'll make the change!